### PR TITLE
feat(cart items): add support for cart items pass through [TA-458]

### PR DESCRIPTION
**Pull request by gryevns**
Adds support for passing an array of cart items via embed.

### CODE REVIEW

File: packages/embed/src/validation.ts [hunk 1]
```diff
@@ -380,4 +380,11 @@ export const validate = (options: SetupConfig) =>
     type: 'object',
     message: 'must be an object',
     required: false,
+  }) &&
+  validateType({
```
[the remark this answers is not in the document]
Comment (douglaseggleton) [reply packages/embed/src/validation.ts:384]: I don't think we need to be super strict at the moment - as long as its an array of data, the API will return an error.

File: packages/embed/src/types.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ export type Config = {
   onCustomSubmit?: ({ method }) => void
   metadata?: Record<string, string>
   paymentSource?: 'installment' | 'recurring'
+  cartItems?: Array<Record<string, unknown>>
```
Comment (douglaseggleton) [inline packages/embed/src/types.ts:30]: I think the interface is important for library consumers - https://github.com/gr4vy/embed-ui/pull/142/files
Embed UI doesn't need to know - but end users using TS will benefit from it.

**Review by douglaseggleton — CHANGES_REQUESTED** [review]
LGTM - I think just needs the typings.

**Review by douglaseggleton — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.6.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.6.0) :rocket:

**Pull request merged**